src/SFA.DAS.Assessor.Functions/Startup.cs

```diff
@@ -56,8 +56,7 @@ public override void Configure(IFunctionsHostBuilder builder)
 
             builder.Services.AddOptions();
             builder.Services.Configure<AssessorApiAuthentication>(config.GetSection("AssessorApiAuthentication"));
-            builder.Services.Configure<SqlConnectionStrings>(config.GetSection("SqlConnectionStrings"));
-            builder.Services.Configure<NotificationsApiClientConfiguration>(config.GetSection("NotificationsApiClientConfiguration"));
+            builder.Services.Configure<SqlConnectionStrings>(config.GetSection("SqlConnectionStrings"));            
```

Comment:
If the notifications api is not being used the nuget package should be removed and any configuration and the actual configuration from the employer-config - there might not be any configuration - it might just be the nuget??